repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/ServiceClient.java

```diff
@@ -80,7 +79,7 @@ public ServiceClient(String targetHostUri, String username, String password, Str
     }
 
     private HttpClient getClientWithBasicAuth() throws ServiceClientException {
```

Comment:
Good call. I don't see the exception being constructed anywhere, so I'll remove it and make sure everything still works.